plugin.program.autowidget/resources/lib/menu.py

```diff
@@ -388,6 +377,8 @@ def show_path(
                 isFolder=not paged_widgets or merged,
                 props=properties,
             )
+            # Ensure we precache next page for faster access
+            cache.cache_expiry(file["file"], widget_id)
```

Comment:
Here's the magic where we pre-cache the next page.

Reply:
I think this means a precached next page is going to cause an unneeded widget redraw. Pass some flag in teh queue file to say not to refresh the widget if its a precache?